access-checkout/src/test/java/com/worldpay/access/checkout/validation/decorators/CvcFieldDecoratorTest.kt

```diff
@@ -116,16 +114,14 @@ class CvcFieldDecoratorTest {
         given(cvcEditText.filters).willReturn(emptyArray())
         cvcFieldDecorator.decorate()
 
-        verify(cvcEditText).setInputType(InputType.TYPE_CLASS_NUMBER)
+        verify(cvcEditText).inputType = InputType.TYPE_CLASS_NUMBER
```

Comment:
Is that triggering mockito's verification process? Isn't that just assigning a value to the inputType property? (same comment for other classes below)

Reply:
double checked this, its still triggering mockitos verification. Also changed the line to 

`verify(cvcEditText, times(1)).inputType = InputType.TYPE_CLASS_NUMBER`

and tested with different inputType(s) to see the expected result